dvc/logger.py

```diff
@@ -102,27 +102,23 @@ def _description(self, message, exception):
 
     def _walk_exc(self, exc_info):
         exc = exc_info[1]
-        tb = traceback.format_exc(*exc_info)
 
         exc_list = [str(exc)]
-        tb_list = [tb]
 
-        # NOTE: parsing chained exceptions. See dvc/exceptions.py for more info
         while hasattr(exc, "__cause__") and exc.__cause__:
```

Comment:
Again, what about `__context__`?

Reply:
@Suor , `__context__`? We are not using `__context__`, and that should be the matter of another PR, I guess

Reply:
Python 3 automatically assigns `__context__` when you raise an exception while handling exception. So if you don't use either `from exc` or `from None` you still can show the original cause, this is usually an error in an error handler, which we probably want to see.